Kconfig

```
@@ -195,6 +195,14 @@ config SIDEWALK_BLE_ADV_INT_SLOW
 	range 20 10240
 	default 1000
 
+config SIDEWALK_BLE_ID
+	int "Set Sidewalk's BLE identity identifier"
+	range 1 255
```

I'm curious is it possible to set  `SIDEWALK_BLE_ID` to e.g 32 when `BT_ID_MAX` is set to 2?
May be here should be something like:
```suggestion
	range 1 BT_ID_MAX
```

---

Due to the implementation of BT stack BT_ID_MAX has to be higher than a max number of needed IDs, e.q. to use 2 IDs we need to set BT_ID_MAX to at least 3.

I have added a BUILD_ASSERT to compare these values